ngx-fudis/projects/ngx-fudis/src/lib/utilities/storybook.ts

```diff
@@ -71,14 +71,12 @@ const buttonCommonExclude: string[] = [
   'dropdownMenuId',
   'dropdownOpen',
   'buttonEl',
-  'handleBlur',
   'buttonClick',
   'closeMenu',
   'toggleMenu',
   'popoverTriggerLabel',
   'popoverPosition',
   'popoverText',
-  'id',
```

Comment:
I think your intention here has been to add `handleBlur` and `id` to the Button Properties list, which is correct. But now these properties have been added to controls view as well. So you still need to add them to `buttonControlsExclude` and `buttonIconOnlyExclude` to exclude them from the controls view.